packages/utilities/src/index.ts

```diff
@@ -27,8 +26,28 @@ export * from './autobind';
 export * from './classNamesFunction';
 export * from './createRef';
 export * from './css';
-export * from './customizable';
-export * from './dom';
+export * from './customizations/Customizations';
+export * from './customizations/Customizer';
+export * from './customizations/Customizer.types';
+export * from './customizations/CustomizerContext';
+export * from './customizations/customizable';
+export * from './customizations/mergeCustomizations';
+export * from './customizations/mergeSettings';
+export * from './dom/IVirtualElement';
```

Comment:
Do we want a `dom/index` module?

Reply:
I am not sure it matters. And in fact, sometimes it ends up hading bugs. For example, if there were a leftover `import { thing } from './dom'`, it would still work since webpack would resolve it (but I wrote a script to validate imports are files and not folders, which would catch it.)